test/extended/openstack/loadbalancers.go

```diff
@@ -87,83 +90,88 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack][lb][Serial] The O
 	for _, i := range availableLbProvidersUnderTests {
 		lbProviderUnderTest := i
 
-		// https://issues.redhat.com/browse/OSASINFRA-2753
-		g.It(fmt.Sprintf("should create an UDP %s LoadBalancer when an UDP svc with type:LoadBalancer is created on Openshift", lbProviderUnderTest), func() {
+		for _, j := range availableProtocolsUnderTests {
+			protocolUnderTest := j
 
-			skipIfNotLbProvider(lbProviderUnderTest, cloudProviderConfig)
+			// https://issues.redhat.com/browse/OSASINFRA-2753
+			// https://issues.redhat.com/browse/OSASINFRA-2412
+			g.It(fmt.Sprintf("should create an %s %s LoadBalancer when an %s svc with type:LoadBalancer is created on Openshift", protocolUnderTest, lbProviderUnderTest, protocolUnderTest), func() {
```

Comment:
nit: `s/when an %s svc/when a %s svc/`, which works for both UDP and TCP

Reply:
done